packages/uniforms/__suites__/ListField.tsx

```diff
@@ -0,0 +1,106 @@
+import { screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import React, { ComponentType, FC } from 'react';
+
+import createContext, { render } from './renderWithContext';
+
+function getDefaultContext() {
+  return createContext({ x: { type: Array }, 'x.$': { type: String } });
+}
+
+function defaultAddFieldLocator(): HTMLElement | null | undefined {
+  return screen.getAllByRole('button')[0];
+}
+export function runListFieldTests(
+  ListField: ComponentType<any>,
+  addFieldLocator = defaultAddFieldLocator,
```

Comment:
1. Let's put it in an object.
2. I'd rather not have a default one at all.

Reply:
Ad. 1. - do you suggest also changing the API for text field suite even if it only accepts one argument?

Reply:
No no, wrap just the options.